README.md

```diff
@@ -64,7 +64,7 @@ Example output:
 ![](resources/biaslyze-demo-box-plot.gif)
 
 
-See more detailed examples in the [tutorial](tutorials/tutorial-toxic-comments/).
+See more detailed examples in the [tutorial](notebooks/notebooks/Tutorial_biaslyze_toxic-comments-usecase.ipynb).
```

Comment:
The issue here is, that the Readme is also used as the Landingpage of the website, so different links are needed there.
A solution could be to link to `https://biaslyze.org/tutorials/tutorial-toxic-comments/` which would work in both settings.